src/java/org/apache/cassandra/net/AsyncStreamingOutputPlus.java

```diff
@@ -170,36 +170,50 @@ public int writeToChannel(Write write, StreamRateLimiter limiter) throws IOExcep
      */
```

Comment:
nit: If we fix up the JavaDoc at all, I would use `link` instead of `code`, given it makes in IDE navigation easier.